packages/epo-widget-lib/src/charts/Base/index.tsx

```diff
@@ -28,7 +28,10 @@ const BaseChart: FunctionComponent<PropsWithChildren<BaseChartProps>> = ({
   return (
     <Styled.ChartContainer>
       {verticalLabel && (
-        <Styled.VerticalLabel id={verticalLabelId}>
+        <Styled.VerticalLabel
+          id={verticalLabelId}
+          aria-hidden={!!verticalLabelId}
```

Comment:
hide these from A11y tree if they are being used as labels for an axis (so they are not read twice)